core/src/main/java/com/datastax/oss/driver/internal/core/control/ControlConnection.java

```diff
@@ -501,6 +515,20 @@ private void forceClose() {
     }
   }
 
+  private boolean isAuthFailure(Throwable error) {
+    boolean authFailure = true;
+    if (error instanceof AllNodesFailedException) {
+      Collection<Throwable> errors = ((AllNodesFailedException) error).getErrors().values();
```

Comment:
if `getErrors()` will return empty connection we will treat it as the `AuthFailure`. Is it ok for us? Is it possible?